tests/autoria/test_average.py

```diff
@@ -0,0 +1,51 @@
+import requests_mock
+import json
+
+from autoria.api import RiaAverageCarPrice
+
+
+class TestAverage:
+    """Tests for autoria average car price calculation."""
+
+    def test_average_minimal(self, ria_categories, ria_marks, ria_states,
+                             ria_bodystyles, ria_models, ria_cities,
+                             ria_gearboxes, ria_options, ria_fuels,
+                             ria_driver_types, ria_colors, ria_average):
+        """Car price can be calculated with minimal set of parameters."""
+        with requests_mock.Mocker() as mock:
```

Comment:
This whole thing with request mocks can be moved into separate fixture (actually should be if we will add more tests). For now I don't do it to keep it simple for start.